roles/os_env_extras_node/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- copy: src=enter-container.sh dest=/usr/local/bin/enter-container.sh mode=0755
```

Comment:
Consider it axed, thanks...  Do we still need the root user as part of the dockerroot group? As far as I can tell it is meant for giving non-root users access to the docker daemon. @jwhonce do you know?